apps/backend/config/app_config.ts

```diff
@@ -69,6 +69,15 @@ export default class AppConfig {
     };
   }
 
+  getDbURL() {
+    let url = this.get('DATABASE_URL');
+    if (url === undefined) {
+      const dbConfig = this.getDbConfig();
+      url = `pg://${dbConfig.username}:${dbConfig.password}@${dbConfig.host}:${dbConfig.port}/${dbConfig.database}`
+    }
+    return url
+  }
```

Comment:
I would prefer if we didn't have to have a function to figure out the DATABASE_URL and could instead standardize on using the `getDbConfig` object everywhere if it is supported.